### DISCUSSION TITLE
async-neocities has a bin

### DISCUSSION THREAD

**Discussion by giscus** [bcomnes/bret.io#226] [Announcements]
# async-neocities has a bin

A running log of announcements, projects and accomplishments.

https://bret.io/blog/2024/async-neocities-bin/

**Comment by BlueChinchompa** [comment]
ooo this will be useful for me thanking you for making this post

**Reply by bcomnes** [reply]
Arg sorry for the delayed response. Great! I hope it works for your use case. Let me know if you run into any issues. Seems like the updated version is working well for most people.
